### PR TITLE
change entities lookup method

### PR DESCRIPTION
**Pull request by lpawelcz**
We want to change the way that entities are looked up to reduce nodes number in final result. We are looking for entities with tag "nam_liv" which is above 0. For simple preview you can revert 739137a8d

### CODE REVIEW

File: server/app/services/textAnalyse.py [hunk 1]
```diff
@@ -83,24 +102,28 @@ def getTextInf(textToSend):
...
     ctag_attr = ccl_ctag(ccl)
+    annot = ccl_ann(ccl)
 
-    return [ccl, bases, poses, ctag_attr]
+    return [ccl, bases, poses, ctag_attr, annot]
 
 
-def tableInit(xml, bases, poses, ctag_attr, weight):
+def check_entity(annot, ctag, base, arr, cnt):
+    if ((annot > 0) and (("subst" in ctag and "sg" in ctag) or ("ign" in ctag))):
```
Comment (wojtasiq) [inline server/app/services/textAnalyse.py:111]: cut this line?

Comment (wojtasiq) [inline server/app/services/textAnalyse.py:111]: Cut this line?

Comment (lpawelcz) [reply server/app/services/textAnalyse.py]: sure, it is done

**Comment by lpawelcz** [comment]
@Isild check it plz

**Pull request merged**